test/numerical/TestGRU.cpp

```diff
@@ -52,11 +52,22 @@ int main(int argc, char *argv[]) {
       argc, argv, "TestGRU\n", nullptr, "TEST_ARGS");
   std::cout << "Target options: \""
             << getCompilerOption(OptionKind::TargetAccel) << "\"\n";
+  // Get configurations from an environment variable
+  std::map<std::string, std::string> opts =
+      ModelLibBuilder::getTestConfigFromEnv("TEST_CONFIG");
+  // Set configuration for test
+  int minL;
```

Comment:
please set the default here, and remove the 'else` clauses, tx.

Reply:
Done. Thanks!